version/version.go

```diff
@@ -1,26 +1,32 @@
 package version
 
 import (
+	"flag"
 	"fmt"
 	"os"
 )
 
-// BuildDate is the date when the binary was built
-var BuildDate string
+var (
+	// BuildDate is the date when the binary was built
+	BuildDate string
+	// GitCommit is the commit hash when the binary was built
+	GitCommit string
+	// MICVersion is the version of the MIC component
+	MICVersion string
+	// NMIVersion is the version of the NMI component
+	NMIVersion string
 
-// GitCommit is the commit hash when the binary was built
-var GitCommit string
-
-// MICVersion is the version of the MIC component
-var MICVersion string
-
-// NMIVersion is the version of the NMI component
-var NMIVersion string
+	// custom user agent to append for adal and arm calls
+	customUserAgent = flag.String("custom-user-agent", "", "User agent to append in addition to pod identity component and versions.")
```

Comment:
Is `flag.Parse()` not necessary, or it's handled in `main.go`?

Reply:
It's handled in `main.go`.